src/DataFixtures/HobbyFixtures.php

```diff
@@ -9,13 +9,28 @@
 
 class HobbyFixtures extends Fixture implements DependentFixtureInterface
 {
-    public const HOBBIES = ['Equitation', 'Dessin', 'Danse classique'];
+    public const HOBBIES = [
+        [
+            'name' => 'Equitation',
+            'candidat' => 'candidat_1'
+        ],
+        [
+            'name' => 'Danse classique',
+            'candidat' => 'candidat_2'
+        ],
+
+        [
+            'name' => 'Dessin',
+            'candidat' => 'candidat_3'
+        ]
+    ];
+
     public function load(ObjectManager $manager): void
     {
         foreach (self::HOBBIES as $hobby) {
             $newHobby = new Hobby();
-            $newHobby->setName($hobby);
-            $newHobby->setCandidat($this->getReference('candidat_john'));
+            $newHobby->setName($hobby['name']);
+            $newHobby->setCandidat($this->getReference($hobby['candidat']));
```

Comment:
tu peux faire plus simple. 
tu fais ton tableau HOBBIES = ['Equitation', 'Danse', 'Dessin', ...] 
et tu boucles avec foreach(self::HOBBIES as $key => $hobby) 
et donc setName($hobby) et setCandidat($this->getReference('candidat'. ($key+1));

Reply:
ou autre solution sans doute encore plus simple si le random te gènes pas (mais je pense pas dans ce cas) `setCandidat($this->getReference('candidat'. rand(0, CandidateFixutures::MAX_CANDIDAT);`